### DISCUSSION TITLE
Is it runnable on MacOS?

### DISCUSSION THREAD

**Discussion by Ayasono** [Equim-chan/mjai-reviewer#66] [Q&A] [answered]
### I have been keeping failing running ako-chan, I am not sure am I doing the right thing...

**Comment by Equim-chan** [comment, marked as answer]
It is, check the readme.

**Reply by Ayasono** [reply]
Great! I have tried it, nice job! If you need a front-end engineer, please feel free to reach out to me, I'm more than glad to help.
